Misc/NEWS.d/next/Library/2019-04-28-01-52-39.bpo-26978.Lpm-SI.rst

```diff
@@ -0,0 +1,2 @@
+`pathlib.path.link_to()` is now implemented. It creates a hard link pointing
```

Comment:
It should be `pathlib.Path.link_to()`, not `pathlib.path.link_to()`.

Also, double backqoutes should be used here instead of single backquotes (as per https://devguide.python.org/documenting/#inline-markup). The following could also be used:

```rst
:meth:`pathlib.Path.link_to()`
```